repo2docker/buildpacks/julia.py

```diff
@@ -22,9 +39,29 @@ def get_env(self):
         ]
 
     def get_path(self):
-        return super().get_path() + ['${JULIA_PATH}/bin']
+        """Adds path to Julia binaries to user's PATH.
+
+         Returns:
+             an ordered list of path strings. The path to the Julia
+             executable is added to the list.
+
+        """
+        return super().get_path() + ['${JULIA_HOME}']
 
     def get_build_scripts(self):
+        """
+        Return series of build-steps common to "ALL" Julia repositories
+
+        All scripts found here should be independent of contents of a
+        particular repository.
+
+        This installs:
+
+        - the specified Julia version and path
+        - Julia packages with ownership granted to the notebook user
```

Comment:
It only creates a directory with permissions for installing julia packages (from get_assemble_scripts), does not actually install them here

Reply:
Thanks. I will update. Still working on the corrections from yesterday.